### PR TITLE
fix: catch file stream error

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

**Comment by coderhaoxin** [comment]
cc @fengmk2

**Comment by coderhaoxin** [comment]
https://github.com/cojs/busboy/issues/28
https://github.com/cojs/busboy/pull/29

**Comment by imyelo** [comment]
any progress on this?

**Comment by coderhaoxin** [comment]
@fengmk2 any advice?

File: index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = function (request, options) {
   .on('field', onField)
   .on('file', onFile)
   .on('close', cleanup)
-  .on('error', onEnd)
+  .on('error', onError)
   .on('finish', onEnd)
```
Comment (fengmk2) [inline index.js]: `finish` event will be emit after`error` event?

File: index.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ module.exports = function (request, options) {
   .on('field', onField)
   .on('file', onFile)
   .on('close', cleanup)
-  .on('error', onEnd)
+  .on('error', onError)
```
Comment (fengmk2) [inline index.js]: cleanup should change too.

File: index.js (hunk 1 above)
Comment (coderhaoxin) [inline index.js]: yea

File: index.js (hunk 2 above)
Comment (coderhaoxin) [inline index.js]: done

**Comment by fengmk2** [comment]
2.0.1

**Pull request merged**